Tests/CryptoTests/Utils/PrettyBytesTests.swift

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCrypto open source project
+//
+// Copyright (c) 2019-2020 Apple Inc. and the SwiftCrypto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import Foundation
+import XCTest
+
+// Not sure if NSString-bridged "String.init(format:_:)" is available on Linux/Windows.
+#if (os(macOS) || os(iOS) || os(tvOS) || os(watchOS)) && CRYPTO_IN_SWIFTPM_FORCE_BUILD_API
```

Comment:
Sorry, this line needs to be:

```
#if (os(macOS) || os(iOS) || os(watchOS) || os(tvOS)) && CRYPTO_IN_SWIFTPM && !CRYPTO_IN_SWIFTPM_FORCE_BUILD_API
// Skip tests that require @testable imports of CryptoKit.
#else
#if (os(macOS) || os(iOS) || os(watchOS) || os(tvOS)) && !CRYPTO_IN_SWIFTPM_FORCE_BUILD_API
@testable import CryptoKit
#else
@testable import Crypto
#endif
```

Reply:
I did look at what other tests wrote. But I wasn't checking whether to use `CryptoKit` or `Crypto`, rather I needed to know (guess) whether `String.init(format:_:)` was available in the SDK. On appleOS this API is provided for sure by bridging from `NSString`'s initializer, but on Linux and Windows I couldn't know.

The code you suggested would make this test always available cross-platform. Let me verify if Swift's cross-platform `Foundation` port has this API. Otherwise I'll need to do something like `sprintf` to make it cross-platform.

Reply:
Also `CryptoTests` defines its own copy of `PrettyBytes`. It's weird because I though `@testable import` means that test target has access to everything non-private in the original `Crypto` module.

This would mean the `PrettyBytesTests` does not actually need to import anything, as the test only tests functions defined in the test target itself.

Can you clarify this?

Reply:
I'm happy with only testing the one in the tests module, tbh.